src/sentry/dynamic_sampling/feature_multiplexer.py

```diff
@@ -57,13 +57,11 @@ def get_user_biases(user_set_biases: Optional[List[Bias]]) -> List[Bias]:
                 returned_biases.append(bias)
         return returned_biases
 
+    @classmethod
+    def get_enabled_user_biases(cls, user_set_biases: Optional[List[Bias]]) -> Set[str]:
```

Comment:
```suggestion
    def get_active_user_biases(cls, user_set_biases: Optional[List[Bias]]) -> Set[str]:
```
just to project bias["active"]. But up to you.